pkg/cli/push.go

```diff
@@ -49,6 +49,16 @@ func push(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("To push images, you must either set the 'image' option in cog.yaml or pass an image name as an argument. For example, 'cog push r8.im/your-username/hotdog-detector'")
 	}
 
+	replicatePrefix := fmt.Sprintf("%s/", global.ReplicateRegistryHost)
+	if strings.HasPrefix(imageName, replicatePrefix) {
+		err := ManifestInspect(imageName)
+		if err != nil {
+			if strings.Contains(err.Error(), `"code":"NAME_UNKNOWN"`) {
+				return err
```

Comment:
Wrap user-facing error with human readable message. Something like this:

```suggestion
				return fmt.Errorf("Unable to find Replicate existing model for %s. Go to replicate.com and create a new model before pushing.", imageName)
```